src/main/kotlin/lotto/domain/Tickets.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+class Tickets(private val _tickets: List<Lotto>) {
```

Comment:
```suggestion
class Tickets(_tickets: List<Lotto>) {
```
프러퍼티로 있을 필요 없어 보이네요.